src/System.Memory/ref/System.Memory.cs

```diff
@@ -91,13 +72,20 @@ public partial struct Span<T>
         public bool TryCopyTo(System.Span<T> destination) { throw null; }
     }
 
-    public static class SpanExtensions
+    public static class Span
```

Comment:
Is this a case where we'd still want to bump the version? If we shipped something at all before (which I'm not sure about), then perhaps the version should be bumped.